jupyterhub/jupyterhub/overlays/build/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ bases:
 - ../../base
 resources:
```

Comment:
For the `build` overlay, you will need to patch the DeploymentConfig to point to the `jupyterhub-img:latest` ImageStream otherwise it will still point to the `quay.io/odh-jupyterhub/jupyterhub-img:v0.2.8` image

Reply:
Ok, first patch, I've applied a replace operation in the image to point to the IS image, I hope this is the correct format.